.NET/Microsoft.Recognizers.Text.DateTime/Spanish/Extractors/SpanishDateExtractorConfiguration.cs

```diff
@@ -151,10 +162,38 @@ public SpanishDateExtractorConfiguration()
 
         IEnumerable<Regex> IDateExtractorConfiguration.ImplicitDateList => ImplicitDateList;
 
+        IImmutableDictionary<string, int> IDateExtractorConfiguration.DayOfWeek => DayOfWeek;
+
         Regex IDateExtractorConfiguration.OfMonth => OfMonth;
 
         Regex IDateExtractorConfiguration.MonthEnd => MonthEnd;
 
         Regex IDateExtractorConfiguration.DateUnitRegex => DateUnitRegex;
+
+        Regex IDateExtractorConfiguration.ForTheRegex => ForTheRegex;
+
+        Regex IDateExtractorConfiguration.WeekDayAndDayOfMothRegex => WeekDayAndDayOfMothRegex;
+
+        private static ImmutableDictionary<string, int> InitDayOfWeek()
```

Comment:
Why was this added here now?

Reply:
1. The English DayOfWeek is configured in Yaml file but the Spanish are not.
2. For "the Friday the 8th", it has to check the 8th in this month is Friday or not. If yes, it will extract as a whole. If no, it will extract two entities "the Friday" and "the 8th".